### PR TITLE
RES-006-contact-page 

### PR DESCRIPTION
**Pull request by EMEZYK**

### CODE REVIEW

**Review by RafalPe — DISMISSED** [review]
Seems fine

File: public/index.html [hunk 1]
```diff
@@ -9,12 +9,26 @@
...
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap"
+      rel="stylesheet"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Rufina&display=swap"
+      rel="stylesheet"
+    />
```
Comment (lagdush) [inline public/index.html]: this should be in one import from google fonts.

File: src/components/ContactPage/ContactPage.js [hunk 2]
```diff
@@ -1,5 +1,106 @@
...
+              style={{
+                textTransform: "uppercase",
+                fontSize: "clamp(1.2rem, 3vw, 2.4rem)",
+                fontFamily: "Rufina",
+              }}
+            >
+              Opening hours
+            </Card.Title>
+            <Card.Text className="mt-3">Mon-Sun 12am to 10pm</Card.Text>
+            <img
```
Comment (lagdush) [inline src/components/ContactPage/ContactPage.js:43]: Where is alt?

**Review by lagdush — CHANGES_REQUESTED** [review]
Zrzut ekranu 2022-01-22 o 14 14 37
Work on the layout and responsiveness.

**Review by lagdush — CHANGES_REQUESTED** [review]
Zrzut ekranu 2022-01-24 o 18 18 36
Still needs improvement

**Review by lagdush — CHANGES_REQUESTED** [review]
Zrzut ekranu 2022-01-24 o 22 10 04
Zrzut ekranu 2022-01-24 o 22 10 12
There is still room for improvement.

**Review by lagdush — APPROVED** [review]

**Review by MateuszWyborski — APPROVED** [review]
Looking good

**Pull request merged**